IPython/html/base/handlers.py

```diff
@@ -138,6 +138,10 @@ def base_url(self):
         return self.settings.get('base_url', '/')
 
     @property
+    def default_url(self):
+        return self.settings.get('default_url', '/')
```

Comment:
Should the fallback here be something other than `/` to avoid an infinite redirect loop?